packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+$accordion-item-padding: 16px;
+$accordion-item-medium-border-radius: 6px;
+$accordion-gap: 12px;
```

Comment:
the fact that the vertical spacing between items and the spacing between the content and the chevron button have the same value, it's coincidental. what I suggest to to is to remove this variable and use directly the value in the two declarations below